src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -383,38 +386,46 @@ public void onTraversalStart() {
         }
 
         // Check for reference version (in)compatibility:
-        if ( allowHg19ContigNamesWithB37 &&
-             referenceVersion.equals(FuncotatorArgumentDefinitions.HG19_REFERENCE_VERSION_STRING) ) {
+        if ( forceB37ToHg19ContigNameConversion || referenceVersion.equals(FuncotatorArgumentDefinitions.HG19_REFERENCE_VERSION_STRING) ) {
             // NOTE AND WARNING:
             // hg19 is from ucsc. b37 is from the genome reference consortium.
             // ucsc decided the grc version had crap in it, so they blocked out some of the bases, aka "masked" them
-            // so the lengths of the contigs are the same, the bases are just _slightly_ different
-            inputReferenceIsB37 = FuncotatorUtils.isSequenceDictionaryUsingB37Reference(getSequenceDictionaryForDrivingVariants());
+            // so the lengths of the contigs are the same, the bases are just _slightly_ different.
+            // ALSO, the contig naming convention is different between hg19 and hg38:
+            //      hg19 uses contigs of the form "chr1"
+            //      b37 uses contigs of the form  "1"
+            // This naming convention difference causes a LOT of issues and was a bad idea.
+            inputReferenceIsB37 = forceB37ToHg19ContigNameConversion || FuncotatorUtils.isSequenceDictionaryUsingB37Reference(getSequenceDictionaryForDrivingVariants());
             if ( inputReferenceIsB37 ) {
                 logger.warn("WARNING: You are using B37 as a reference.  " +
                         "Funcotator will convert your variants to GRCh37, and this will be fine in 99.9% of cases.  " +
```

Comment:
Can you say "vast majority"?  "99.9%" makes it seem like you really studied this and derived an actual number....

Reply:
Sure.